scripts/run_examples

```diff
@@ -46,12 +41,14 @@ done
 examples="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/'"${application_language}"'$' | cut -d'/' -f2 | uniq)"
 for example in ${examples}; do
   if [[ "${example}" == 'aggregator' ]]; then
-      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "${client_variants}" -e aggregator -- --bucket=test --data=1:10,2:20,3:30
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -e aggregator -- --bucket=test --data=1:10,2:20,3:30
   elif [[ "${example}" == 'chat' ]]; then
-      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "${client_variants}" -e chat -- --test
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -e chat -- --test
   elif [[ "${example}" == 'trusted_information_retrieval' ]]; then
-      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "${client_variants}" -e trusted_information_retrieval -- --latitude 0.0 --longitude 0.0
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "bazel cargo" -e trusted_information_retrieval -- --latitude 0.0 --longitude 0.0
+  elif [[ "${example}" == 'hello_world' ]]; then
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "bazel npm" -e "${example}"
   else
-      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c "${client_variants}" -e "${example}"
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -e "${example}"
   fi
```

Comment:
Personally I think keeping a kind of manifest is preferable over all these conditionals. 

However given that we still haven't decided on the bash vs rust question of runner scripts, I don't want to put too much effort into refactoring the bash scripts to remove redundancy.

Reply:
I note that (to me) this seems a good argument for the Rust + manifest, so feel free to weigh in on the discussion :) I would like it to be driven by use cases, and this seems a good one to me

Reply:
Agreed. And as [I mentioned earlier](https://github.com/project-oak/oak/pull/1105#discussion_r438030248), replacing these scattered conditionals is what appeals to me most about the manifest approach. 

Though I suppose while Rust makes it easier to maintain complex logic such manifests, one could always argue that theoretically the same pattern could be implemented in bash. :)